Solutions/Menes.Hosting.AspNetCore/Menes/Internal/OpenApiResultOutputBuilder.cs

```diff
@@ -0,0 +1,100 @@
+﻿// <copyright file="OpenApiResultOutputBuilder.cs" company="Endjin Limited">
+// Copyright (c) Endjin Limited. All rights reserved.
+// </copyright>
+
+namespace Menes.Internal
+{
+    using System.Collections.Generic;
+
+    using Menes.Converters;
+
+    using Microsoft.Extensions.Logging;
+    using Microsoft.OpenApi.Models;
+
+    /// <summary>
+    /// Common logic for building a response for an <see cref="OpenApiResult"/>, with output
+    /// validation against the <see cref="OpenApiOperation"/> definition.
+    /// </summary>
+    /// <typeparam name="TResponse">The response type.</typeparam>
+    internal abstract class OpenApiResultOutputBuilder<TResponse> : IResponseOutputBuilder<TResponse>
+    {
+        private readonly IEnumerable<IOpenApiConverter> converters;
+        private readonly ILogger logger;
+
+        /// <summary>
+        /// Creates an <see cref="OpenApiResultOutputBuilder{TResponse}"/>.
+        /// </summary>
+        /// <param name="converters">The open API converters to use with the builder.</param>
+        /// <param name="logger">The logger for the output builder.</param>
+        public OpenApiResultOutputBuilder(
```

Comment:
Shouldn't abstract constructors be protected?

Reply:
Yes, they should. Fixed.